Meta/check-style.py

```diff
@@ -46,6 +47,16 @@
 # LibC is supposed to be a system library; don't mention the directory.
 BAD_INCLUDE_LIBC = re.compile("# *include <LibC/")
 
+# Make sure that all includes are either system includes or immediately resolvable local includes
+ANY_INCLUDE_PATTERN = re.compile('^ *# *include\\b.*[>"](?!\\)).*$', re.M)
+SYSTEM_INCLUDE_PATTERN = re.compile("^ *# *include *<([^>]+)>(?: //.*)?$")
+LOCAL_INCLUDE_PATTERN = re.compile('^ *# *include *"([^>]+)"(?: /[*/].*)?$')
```

Comment:
Don´t you mean `[^"]` here?

Also, why do local includes allow for `/* */` type comments, but system includes do not?

Reply:
You're right! My intention was to remove it, but it really is used for some local includes, and forgot to allow it for system includes.

I'll change it to allow for both.